src/snowflake/cli/api/project/schemas/project_definition.py

```diff
@@ -42,6 +42,11 @@
 from snowflake.cli.api.utils.types import Context
 from typing_extensions import Annotated
 
+AnnotatedEntity = Annotated[Entity, Field(discriminator="type")]
+EntityOrList = Union[
+    AnnotatedEntity, List[AnnotatedEntity]
+]  # TODO: Probablyu should delete this line
+
```

Comment:
Is this change neccessary? Are you building on current main?

Reply:
It's just for simplicity - same thing is used 4 times, so seems reasonable to put it into variable